### PR TITLE
Toggle truncate

### PR DESCRIPTION
**Pull request by pon-y**
Add click functionality to "See More/See Less" line that will expand and hide song details past a certain length.

Please let me know if there are any questions or concerns.

### CODE REVIEW

**Comment by pon-y** [comment]
I've enabled circleCI but have not yet written anything in regards to it. Please ignore the failing check.

**Review by sharmoo — APPROVED** [review]
Clean.

**Review by kevinbece5 — APPROVED** [review]

File: client/src/components/songDetailWidget.jsx [hunk 1]
```diff
@@ -60,13 +62,32 @@ class SongDetailWidget extends React.Component {
   }
 
   expandDescription() {
-    return null;
+    // const { this.proptruncated } = this.props;
+    if (this.props.truncated) {
+      return (
+        <div className="showMoreText" onClick={this.props.toggleTruncate}>
+Show more
+          {'  '}
```
Comment (kevinbece5) [inline client/src/components/songDetailWidget.jsx]: @pon-y use padding here instead of open quotes this method adds clutter to the code

**Review by kevinbece5 — CHANGES_REQUESTED** [review]

File: client/src/components/songDetailWidget.jsx (hunk 1 above)
Comment (pon-y) [reply client/src/components/songDetailWidget.jsx]: Thank you! I've removed the {' '} from the file as per your suggestion

**Review by kevinbece5 — APPROVED** [review]

**Pull request merged**